### ISSUE TITLE
Shader fails to compile with error 99999

### ISSUE THREAD

**Issue by buzmeg** [shader-slang/slang#5558]
Compiled slangc from repository:
```
slangc -v
2024.14.5-16-g91f19bed
```

Ubuntu 24.04.1 LTS

Same shader as #5538 results in:

```
slangc cs_copy_scatter.slang -target spirv -profile spirv_1_6 -stage compute -entry main -o cs_copy_scatter.spv
(0): error 99999: Slang compilation aborted due to an exception of N5Slang13InternalErrorE: unimplemented: Unhandled global inst in spirv-emit:
let  %1	: %2	= getElementPtr(%3, %ii)
```
 
Thanks.

**Comment by csyonghe** [comment]
This should be fixed in #5552.

**Comment by csyonghe** [comment]
Closing. PR #5552 has landed, and fix can be verified here: https://shader-slang.com/slang-playground/?target=SPIRV&code=eJyNU_9vmkAU_52_4sVmCSbGKm0WI52JLdSROmeAde0YuZxwyDkFBofaLf3fdxyg1tZm98MF3vt8eXfv3ZlPAhoRuPn8bXKHLOOHjh6g-1E6ex1_LOK7xP3QNIbXYx0NTXP4iPSJbRq6Bd094tYYj9F0aNu6OUGa8UWfWMbXifVS_xSoNJMyluYeg2mehRpmGP5KwFdOI6aAnzEUp3ROI3UXFcHN5igQhgeB_EJBAV0uUYIZI2mEaOSTrSo9qzu7W56eltnrPAhIWvkGyxizS0i2zsmy3ZOpB7f0kBxn_avfT_iRkBdHGcMRc13ppvosDa_qEw8g8WZqTZrxWmk0lzst6DTfJfmzvdUBq1uwLHHKPCV-xXt13gH4yZt8peCXUbrCc4KCOF1hJjfS-Qz3GjwL5nebbAt1RbsqL2wAJTbb4MQLcdEvLh3lKxamBPuZfDh8rRcjJyoGOD8H8juna7wkvIMshmXs4SXK6B8irWPqw4qrykV7L2DEf_nqg3WPRmmcJ4bWEt2rVwnTbI4TII1mfBS80BbVcDS8AR_t4ELzHSyMjCP_Yr6afIQEVLS8BFIKn4Rwe6se5xaLOvekHhLrEVyted5PnI7b5uNIqessFm6F5LeVpFwkkBsfLteKD3y_DH5GjVbl9tQqBJp7-FF_nOLReHGc-pnLbUpPudMuBuhwqxWO6eJ9yl2lx1FKr1locD_1f7GVn3Dp7LbS7_kfK6RtrQ
